lib/cancan/model_adapters/active_record_5_adapter.rb

```diff
@@ -46,8 +46,6 @@ def sanitize_sql(conditions)
 
           conditions.stringify_keys!
 
-          conditions, binds = predicate_builder.create_binds(conditions)
```

Comment:
Quite honestly, I'm not sure now. At the time I was trying to make sense of the changes in Rails 5 and was digging deep into source code to get things working, specifically around the predicate builder. I got things mostly working but another contributor @dahlgren discovered that removing this line made the tests pass. Sorry I can't provide a better explanation.